psi4/src/read_options.cc

```diff
@@ -2775,214 +2775,238 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
-        /*- Maximum number of geometry optimization steps -*/
-        options.add_int("GEOM_MAXITER", 50);
-        /*- Print all optking parameters. -*/
-        options.add_bool("PRINT_OPT_PARAMS", false);
-        /*- Specifies minimum search, transition-state search, or IRC following -*/
-        options.add_str("OPT_TYPE", "MIN", "MIN TS IRC");
-        /*- Geometry optimization step type, either Newton-Raphson or Rational Function Optimization -*/
-        options.add_str("STEP_TYPE", "RFO", "RFO RS_I_RFO P_RFO NR SD LINESEARCH");
-        /*- Geometry optimization coordinates to use.
-            REDUNDANT and INTERNAL are synonyms and the default.
-            CARTESIAN uses only cartesian coordinates.
-            BOTH uses both redundant and cartesian coordinates.
-            CUSTOM is not fully implemented yet - expected optking 0.3.1  -*/
-        options.add_str("OPT_COORDINATES", "INTERNAL", "REDUNDANT INTERNAL CARTESIAN BOTH CUSTOM");
-        /*- A string formatted as a dicitonary containing a set of coordinates. Coordinates can be
-            appended to Optking's coordinate set or used on their own - expected optking 0.3.1. -*/
+        /* Alphabetized so I can check for missing keywords more easily against sorted model */
+
+        /* */
+        options.add_bool("ACCEPT_SYMMETRY_BREAKING", false);
+        /* Do add bond coordinates at nearby atoms for non-bonded systems? */
+        options.add_bool("ADD_AUXILIARY_BONDS", true);
+        /* Skipping ALG_GEOM_MAXITER */
+        /* This factor times standard covalent distance is used to add extra stretch coordinates. */
+        options.add_double("AUXILIARY_BOND_FACTOR", 2.5);
+        /* SKIPPING BT_X KEYWORDS */
+        /* Do read Cartesian Hessian?  Only for experts - use
+        :py:attr:`~optking.v1.optparams.OptParams.full_hess_every` instead. */
+        options.add_bool("CART_HESS_READ", false);
+        /* */
+        options.add_str("CONJUGATE_GRADIENT_TYPE", "");
+        /* Set number of consecutive backward steps allowed in optimization */
+        options.add_int("CONSECUTIVE_BACKSTEPS", 0);
+        /* When determining connectivity, a bond is assigned if interatomic distance
+        is less than (this number) * sum of covalent radii. */
+        options.add_double("COVALENT_CONNECT", 1.3);
+        /* A string formatted as a dicitonary containing a set of coordinates. Coordinates can be
+            appended to Optking's coordinate set or used on their own - expected optking 0.5 */
         options.add_str("CUSTOM_COORDS", "");
-        /*- Do follow the initial RFO vector after the first step? -*/
-        options.add_bool("RFO_FOLLOW_ROOT", false);
-        /*- Root for RFO to follow, 0 being lowest (for a minimum) -*/
-        options.add_int("RFO_ROOT", 0);
-        /*- Starting level for dynamic optimization (0=nondynamic, higher=>more conservative) -*/
+        /* Starting level for dynamic optimization (0=nondynamic, higher=>more conservative) */
```

Comment:
checking you know that `/*-` and `-*/` are special signals to autodoc option on the following line. fine to remove to consistency from rest, just checking.